code/game/objects/items/ego_weapons/he.dm

```diff
@@ -1747,3 +1747,17 @@
 
 /obj/item/ego_weapon/totalitarianism/get_clamped_volume()
 	return 50
+
+/obj/item/ego_weapon/sunshower //TD
+	name = "sunshower"
+	desc = "I cannot protect you from this rain, but I can guard you from false kindness."
+	icon_state = "sunshower"
+	force = 30
+	attack_speed = 0.5
```

Comment:
This is 60 DPS on an HE weapon, HEs should be around 30. Either curb the attack speed or the damage. Faster attacking weapons do get an advantage in DPS due to requiring lots of clicking and being in constant damage range (as opposed to very slow weapons where you can hit and run). IF you want it fast, go for around 36 DPS, otherwise 30 and no defined attack speed works.